buildSrc/src/main/kotlin/studio/lunabee/library/PublishingExt.kt

```diff
@@ -29,11 +29,8 @@ import org.gradle.api.publish.PublicationContainer
 import org.gradle.api.publish.PublishingExtension
 import org.gradle.api.publish.maven.MavenPublication
 import org.gradle.api.tasks.bundling.Jar
-import org.gradle.kotlin.dsl.create
-import org.gradle.kotlin.dsl.get
-import org.gradle.kotlin.dsl.getValue
-import org.gradle.kotlin.dsl.provideDelegate
-import org.gradle.kotlin.dsl.registering
+import org.gradle.configurationcache.extensions.capitalized
+import org.gradle.kotlin.dsl.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/LunabeeStudio/Lunabee_Compose_Android/blob/eb9aa9e32b25116db742e5197af2f1942311ae96/buildSrc/src/main/kotlin/studio/lunabee/library/PublishingExt.kt#L33"></span>org.gradle.kotlin.dsl.* is a wildcard import. Replace it with fully qualified imports.</td>
    </tr>
  </tbody>
</table>